crates/edr_solidity_tests/tests/testdata/default/cheats/Assert.t.sol

```diff
@@ -6,7 +6,8 @@ import "cheats/Vm.sol";
 
 contract AssertionsTest is DSTest {
     string constant errorMessage = "User provided message";
-    uint256 constant maxDecimals = 77;
+    // For i256
+    uint256 constant maxDecimals = 76;
```

Comment:
Instead of this I'd consider updating our definition of `_formatWithDecimals` to [Foundry's current](https://github.com/foundry-rs/foundry/blob/master/testdata/default/cheats/Assert.t.sol#L48-L53).

Reply:
Thanks, fixed in https://github.com/NomicFoundation/edr/commit/66dde67f649e8150acfad8abd95c99015586abdb